packages/ckeditor5-font-mapper/README.md

```diff
@@ -0,0 +1,133 @@
+# CKEditor Plugin: Font Mapper
+
+The Font Mapper plugin extends the
+[CKEditor 5 Clipboard Feature][cke5:docs:clipboard:feature].
+
+When pasting rich text from external sources into the editor, some characters of
+the pasted text might originate from an unsupported font. This plugin maps such
+characters to their named entities or Unicode equivalents using a mapping table.
+
+By default, this plugin already contains a mapping table for the Word `Symbol` font, and automatically
+converts input content (for instance while pasting from Microsoft Word) accordingly.
+This mapping table can be replaced or extended.
```

Comment:
Questions unanswered here (if I did not oversee something) and may also trigger some additional tests to add:

* Where is the font being looked up? (I think within the `style` attribute `font-family`.)
* Is case important?
* Is the (deprecated) `<font>` element supported?
* What about multiple font-families specified? Actually, we should support it – or document it as limitations for now.

We may also want to address security implications. What, if someone adds a `<script>` tag to the replacement map? This may even happen by some malicious code in an application like CoreMedia Studio which overrides the configuration.

Suggestion: We may want to disallow the `<script>` tag by default and only allow it, when an additional configuration option is given like:

```javascript
// don't do, see below (but keep for brainstorming)
{
  font: "Symbol",
  allowScript: true,
  // ...
}
```

Ok, reverting idea of having this configurable. This will not really lower the risk of malicious code.